client/devicemanager/manager.go

```diff
@@ -18,14 +18,8 @@ import (
 	"github.com/hashicorp/nomad/plugins/shared/loader"
 )
 
-// Manaager is the interface used to manage device plugins
+// Manager is the interface used to manage device plugins
 type Manager interface {
-	// Run starts the device manager
-	Run()
```

Comment:
Is it correct to remove these. This could hamstring the mock implementations/integrators. The managers only actually work if run *is* called

Reply:
I left these removed, but added the `pluginmanager.PluginManager` interface